server/market/market.go

```diff
@@ -1258,6 +1259,19 @@ func (m *Market) Run(ctx context.Context) {
 					currentEpoch.Epoch, epochDuration)
 				// Signal to the book router if this is a resume.
 				if m.suspendEpochIdx != 0 {
+					fmt.Println("********************************************")
+					fmt.Println("hello resume")
+					// Ask storage if lot size changed during
+					// suspend and purge orders if needed.
+					for _, mkt := range m.storage.MarketsNeedOrderPurge() {
+						if mkt != m.marketInfo.Name {
+							log.Warnf("unexpected market found in purgable markets %s", mkt)
+							continue
+						}
+						log.Debugf("Lot size changed, purging orders for mkt %s", m.marketInfo.Name)
+						m.PurgeBook()
+						break // Should not be duplicates, but be safe.
+					}
```

Comment:
Thank you for the great comments.

>  In this PR we just want to deal with dex startup

Oh! The issue does say that. So, suspend persisted, shutdown completely, change markets.json manually, and startup.

Reply:
There sure are a lot of ifs ands or buts in this.

You got the DB update of `lot_size` down just right.

Just needs a little tweak to the purge call if the operator forgets to purge before shutdown. :+1:

Reply:
That reminds me I need to carefully review Swapper functionality w.r.t. lot size (shouldn't be any, but I don't recall), otherwise it's going to be a burden to change lot size with active swaps.

Reply:
Went with your third option, which doesn't require any changes outside of the database code.

Reply:
Actually testing with active swaps, completed swaps, and booked swaps, doesn't seem to cause any errors.